From 143be9ed1029b19a464632aaf45bc6d73fcf2401 Mon Sep 17 00:00:00 2001
From: Parth Pancholi <parth.pancholi@einfochips.com>
Date: Sat, 16 Jan 2016 19:28:43 +0530
Subject: [PATCH 23/40] SD-Card Detect GPIO Logic Changed for DB600C

Signed-off-by: eInfochips Team <ei-qc8064@einfochips.com>
---
 arch/arm/mach-msm/board-8064-storage.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-msm/board-8064-storage.c b/arch/arm/mach-msm/board-8064-storage.c
index 9d8498e7..3f2067e 100644
--- a/arch/arm/mach-msm/board-8064-storage.c
+++ b/arch/arm/mach-msm/board-8064-storage.c
@@ -302,7 +302,11 @@ static struct mmc_platform_data sdc3_data = {
 	.status_gpio	= 26,
 	.status_irq	= MSM_GPIO_TO_INT(26),
 	.irq_flags	= IRQF_TRIGGER_RISING | IRQF_TRIGGER_FALLING,
+#ifdef CONFIG_MACH_APQ8064_DB600C
+	.is_status_gpio_active_low = 0,
+#else
 	.is_status_gpio_active_low = 1,
+#endif
 	.xpc_cap	= 1,
 	.uhs_caps	= (MMC_CAP_UHS_SDR12 | MMC_CAP_UHS_SDR25 |
 			MMC_CAP_UHS_SDR50 | MMC_CAP_UHS_DDR50 |
-- 
1.9.1

